src/query/service/src/interpreters/interpreter_set.rs

```diff
@@ -70,7 +70,12 @@ impl SetInterpreter {
         let mut is_globals = vec![];
 
         for (var, scalar) in self.set.idents.iter().zip(scalars.into_iter()) {
```

Comment:
scalars have been casted into string type, so it's safe to use unsafe here.

Yet the pr is still acceptable.